### PR TITLE
21559 cleanup test0 fixture include test implementations

### PR DESCRIPTION
**Pull request by astares**
https://pharo.fogbugz.com/f/cases/21559/Cleanup-test0FixtureIncludeTest-implementations

### CODE REVIEW

File: src/Collections-Strings/String.class.st [hunk 1]
```diff
@@ -1048,6 +1048,14 @@ String >> endsWithAColon [
 
 ]
 
+{ #category : #comparing }
+String >> endsWithAnyOf: aCollection [
+	aCollection do:[:suffix|
+		(self endsWith: suffix) ifTrue:[^true].
+	].
+	^false
+]
```
Comment (jecisc) [inline src/Collections-Strings/String.class.st]: I think this addition was not intended.

**Review by jecisc — CHANGES_REQUESTED** [review]

File: src/Collections-Strings/String.class.st (hunk 1 above)
Comment (astares) [reply src/Collections-Strings/String.class.st]: This was removed in another PR. Iceberg now added it again ... mmh.

Comment (MarcusDenker) [reply src/Collections-Strings/String.class.st]: it is likely just shown wrong in the diff tool of GitHub

**Comment by astares** [comment]
Mmmh ... iceberg mixes although I start cleanly with a new PR always from dev.

I guess I need to redo this one completely

**Comment by astares** [comment]
OK - so if possible please review again.

**Review by jecisc — APPROVED** [review]

**Comment by astares** [comment]
Thanks!

**Pull request merged**